andela/bp-esa-frontend
# 160819278 Fix dependency security vulnerabilities

#### What does this PR do?
- use `npm audit fix` to fix vulnerabilities from the `url-parse` dependency.
- installed `bootstrap@4.1.2` to replace `bootstrap@4.1.1`.
#### Description of Task to be completed?
- Fix potential security vulnerabilities that exist in the front-end project 
#### How should this be manually tested?
The security vulnerabilities were reported by Github Dependency Graph. To test this fix, one would need to also check the Github Dependency Graph, if the issues are fixed the vulnerabilities reports should be removed.This is a [link](https://help.github.com/articles/listing-the-packages-that-a-repository-depends-on/) on how to check the dependency graph.

**NOTE**: It seems the dependency graph only reports what is in the default branch of the repository, consequently, this pull request would have to be merged to update the reporting of the Github Dependency Graph.
#### What are the relevant pivotal tracker stories?
#160819278

#### Snapshots: 
Github Dependency Graph reporting the vulnerabilities.
![screen shot 2018-09-27 at 5 15 38 pm](https://user-images.githubusercontent.com/26048536/46159802-18573580-c279-11e8-8f27-8a1bb52c2d9c.png)
![screen shot 2018-09-27 at 5 15 56 pm](https://user-images.githubusercontent.com/26048536/46159816-20af7080-c279-11e8-9ffe-5a76ffc1fc0e.png)
More details on `bootstrap` related vulnerabilities:
![screen shot 2018-09-27 at 5 18 53 pm](https://user-images.githubusercontent.com/26048536/46159973-7f74ea00-c279-11e8-931c-09c78dfbbe59.png)
More details on `url-parse` related vulnerabilities:
![screen shot 2018-09-27 at 5 19 05 pm](https://user-images.githubusercontent.com/26048536/46159985-86036180-c279-11e8-8a15-43cac0ebb53e.png)
